app/assets/stylesheets/application.scss

```diff
@@ -105,3 +136,62 @@ body {
   width: 250px;
   padding: 5px;
 }
+
+Masonary
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+body { font-family: sans-serif; }
+
+/* ---- grid ---- */
+
+.grid {
+  background: $mediumaquamarine;
+  max-width: 2000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/* clearfix */
+.grid:after {
+  content: '';
+  display: block;
+  clear: both;
+}
+
+/* ---- grid-item ---- */
+
+.post-item {
+  float: left;
+  margin: 10px;
+  background:rgba(255, 255, 255, 0.5);
+  width: 600px;
+  padding: 20px;
+  height: auto;
+  border: 2px solid #333;
+  border-color: hsla(0, 0%, 0%, 0.5);
+}
+
+.trip-item {
+  float: left;
```

Comment:
Properties should be ordered background, border, border-color, float, height, margin, padding, width